packages/xod-tabtest/src/TabData.re

```diff
@@ -13,12 +13,14 @@ module Value = {
     | Byte(int)
     | Errcode(int)
     | Pulse(bool)
+    | RaisedError(int)
     | Invalid(string);
   let numberRegex = [%re {|/^[+-]?(?=.)*\d*(?:\.\d+)?$/|}];
   let approxNumberRegex = [%re {|/^[+-]?(?=.)*\d*(?:\.\d+)?~$/|}];
   let stringRegex = [%re {|/^".*"$/|}];
   let byteRegex = [%re {|/^[0-9a-f]{2}h|[0,1]{8}b|\d{1,3}d$/i|}];
   let errcodeRegex = [%re {|/^E\d{1,3}$/|}];
+  let raisedErrorRegex = [%re {|/^e\(\d{1,3}\)$/|}];
```

Comment:
Not sure about the syntax. I feel it differs too much from the regular error code. Perhaps a better design would be trailing exclamation: `E42!` What do you think?